### PR TITLE
Add files

### PR DESCRIPTION
**Pull request by Ninjaphil24**
I have completed all the steps, including linters checks on my computer.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,97 @@
...
+## Have you already completed the Sass lessons?
+
+If you completed the Sass lessons you may prefer to work with SCSS files, if that's the case you need to perform
+a few modifications to the linter configuration files.
+
+1. In the file `.github/workflows/linters.yml` [line 48](https://github.com/microverseinc/linters-config/blob/master/html-css/.github/workflows/linters.yml#L48) replace `"**/*.{css,scss}"` with `"**/*.scss"`
+2. Use `npx stylelint "**/*.scss"` to lint you SCSS files and not the generated CSS
+
+Once you've done the previous steps you can focus on your SCSS files and not in fixing the generated CSS output
+every time.
```
Comment (rustam-tolipov) [inline README.md]: ### Hi Ninjaphiil 👋, congratulation on joining the Microverse 🎉🎊.  

- [ ]  Your project is nearly done with no linter errors and any errors in your code. There is one thing that needs to be done and that is your readme, you have used the requirement readme file. Please kindly click on this [link 👉](https://github.com/microverseinc/readme-template) and use this template. I would recommend changing the readme file as you needed like add your own projects screenshot, use your name, and so on 🍀

**Review by rustam-tolipov — CHANGES_REQUESTED** [review]
Hi @Ninjaphil24 , 

Good job so far!

### HIGHLIGHTS!

- No linters errors.💯
- Descriptive Readme.md. 💯


There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Ninjaphil24) [reply README.md]: Hello!  Thanks for the help.  Yes, I completely forgot to do the README.md file.  It's done now.

**Review by julie-ify — APPROVED** [review]
## Hi @Ninjaphil24, 

# Highlights
- Good job implementing the required changes from the previous reviewer ✔️ 

# Status: Approved ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Approved emoji

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
********

**Pull request merged**